plugins/woocommerce/includes/wc-account-functions.php

```diff
@@ -164,6 +164,33 @@ function wc_get_account_menu_item_classes( $endpoint ) {
 	return implode( ' ', array_map( 'sanitize_html_class', $classes ) );
 }
 
+/**
+ * Get current page in order to set the current menu item.
+ *
+ * @param string $endpoint Endpoint.
+ * @return string
+ * @since 9.2.0
```

Comment:
Remember to update the since tags.

Reply:
Thanks @mikejolley for the suggestion, this completely makes sense and I have updated the code to be cleaner. Front End Engineer attempting to do some basic PHP 😬

Reply:
Done ✅